src/Drivers/NexmoSMS.php

```diff
@@ -225,4 +226,11 @@ public function receive($raw)
 
         return $incomingMessage;
     }
+    
+    private function setEncoding()
```

Comment:
`setEncoding` should take an optional encoding parameter. This would allow setting the encoding per individual SMS.